module-system/module-implementations/sov-bank/src/lib.rs

```diff
@@ -5,24 +5,35 @@ mod query;
 mod token;
```

Comment:
Let's add the following add notations:
```
#![deny(missing_docs)]
#![doc = include_str!("../README.md")]
```
So it will fail compilation of public doc is missing and ad README as crate level doc.